nixos/modules/misc/ids.nix

```diff
@@ -628,6 +629,7 @@ in
       restic = 291;
       openvpn = 292;
       # meguca = 293; # removed 2020-08-21
+      httpfs = 293;
```

Comment:
Can we use dynamic users?

Reply:
Hm... probably. unlike the other Hadoop services, it only has a temporary local storage that can be discarded. I'll make changes in prestart to do that and see how it works.

Reply:
Looks like even that won't be necessary because of privateTmp